src/Components/ListArticles.jsx

```diff
@@ -48,71 +45,78 @@ export const ListArticles = ({ query }) => {
+              <option value="&order=asc">
+                Ascending
+              </option>
+              <option value="&order=desc">
+                Descending
+              </option>
+            </Form.Select>
 
-        <Form.Select aria-label="Default select" onChange={handleSelectOrderBy}>
-          <option>Value to sort by</option>
-          <option value="&sort_by=title">
-            Title
-          </option>
-          <option value="&sort_by=author">
-            Author
-          </option>
-          <option value="&sort_by=created_at">
-            Date Created
-          </option>
-          <option value="&sort_by=votes">
-            Votes
-          </option>
-          <option value="&sort_by=comment_count">
-            Comments Count
-          </option>
-        </Form.Select>
+            <Form.Select aria-label="Default select" onChange={handleSelectOrderBy}>
+              <option>Value to sort by</option>
+              <option value="&sort_by=title">
+                Title
+              </option>
+              <option value="&sort_by=author">
+                Author
+              </option>
+              <option value="&sort_by=created_at">
+                Date Created
+              </option>
+              <option value="&sort_by=votes">
+                Votes
+              </option>
+              <option value="&sort_by=comment_count">
+                Comments Count
+              </option>
+            </Form.Select>
 
-        <Button variant="outline-dark buttons" type="submit">Click To Sort</Button>
-      </Form>
-      <h1>Articles</h1>
+            <Button variant="outline-dark buttons" type="submit">Click To Sort</Button>
+          </Form>
+          <h1>Articles</h1>
 
 
-      <Row xs={1} md={2} lg={3} className="g-4">
-        {articles.map((article, index) => {
-          const formattedDate = new Date(article.created_at).toLocaleString();
-          return (
-            <Col key={article.id || index}>
-              <Card>
-                <Card.Img variant="top" src={article.article_img_url}
-                          style={{width: '100%', height: '200px', objectFit: 'cover'}}/>
-                <Card.Body>
-                  <Card.Title>{article.title}</Card.Title>
-                </Card.Body>
-                <ListGroup className="list-group-flush">
-                  <ListGroup.Item>Author: {article.author}</ListGroup.Item>
-                  <ListGroup.Item>Topic: {article.topic}</ListGroup.Item>
-                  <ListGroup.Item>Created: {formattedDate}</ListGroup.Item>
-                  <ListGroup.Item>Votes: {article.votes}</ListGroup.Item>
-                  <ListGroup.Item>Comments: {article.comment_count}</ListGroup.Item>
-                </ListGroup>
-                <Card.Body>
+          <Row xs={1} md={2} lg={3} className="g-4">
+            {articles.map((article, index) => {
+              const formattedDate = new Date(article.created_at).toLocaleString();

```

Comment:
I'd recommend extracting some of this out, for example into a filter component and an articles list component - just a lot going on here that's hard to parse!